ui/round/src/clock/clockView.ts

```diff
@@ -99,8 +99,12 @@ function renderBerserk(ctrl: RoundController, color: Color, position: Position)
 function goBerserk(ctrl: RoundController) {
   if (!game.berserkableBy(ctrl.data)) return;
   if (ctrl.goneBerserk[ctrl.data.player.color]) return;
+
+  var berserkMssg = "GO BERSERK! Half the time, bonus point";
+  const increment = ctrl.data.clock!.increment;
+  if (increment > 0) berserkMssg = "GO BERSERK! Lose increment for a bonus point";
```

Comment:
It feels unnecessary to me. Also the variable is misspelled and possibly assigned twice. And the second message is incorrect, time is still halved.